src/views/Player.vue

```diff
@@ -60,14 +64,17 @@ if (!Array.prototype.indexOf)
 // in milliseconds
 var interval_time = 50;
 
+// How precisely should the question pop-up logic
+// be measured. Time in milliseconds
+var popUpPrecisionTime = 10;
+
```

Comment:
earlier we were relying on the `timeupdate` event provided by `plyr` but that only repeated every `50ms`. But it was not enough to handle those questions which were very close (< `500ms` difference).

Added another `setInterval` and the repetition time of that `setInterval` is this `popUpPrecisionTime `